README.md

```diff
@@ -134,6 +134,10 @@ rust({
 
     // Allows you to customize the behavior for loading the .wasm file, this is for advanced users only!
     importHook: function (path) { return JSON.stringify(path); },
+
+    // Allows you to define a custom path to wasm-pack.
+    // Use of a '~' prefix will be converted to `os.homedir()`
+    wasmPackPath: "~/.cargo/bin/wasm-pack"
```

Comment:
It actually defaults to `".node_modules/.bin/wasm-pack"`

Reply:
Was this just a comment to that effect?

Reply:
The README shows the default values, so it has to be changed.